docs/app/img/icons/octicon-repo.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 48 48">

```

Comment:
You will need to generate the asset cache for this example to work in codepen.  https://github.com/angular/material/blob/master/docs/guides/CODEPEN.md